src/components/BaseCard.vue

```diff
@@ -174,7 +174,9 @@ export default {
     },
     resolvedRq() {
       if (this.car && this.car.selectedTune && this.car.selectedTune.substr(0,3) === 'v15') {
```

Comment:
Btw, @TiagoXavi, maybe it's just better to do
`if (this.car && this.car.selectedTune && this.car.selectedTune.startsWith('v')) {
    return Vue.getOldCar(this.car.rid, this.car.selectedTune.substr(1,3)).rq;`